src/interfaces/store-interfaces/action-interface.ts

```diff
@@ -1,10 +1,7 @@
+// I don't think this file is used any more - CRG Jul23/23
```

Comment:
I felt nervous deleting a file from a code base I'm unfamiliar with, but AFAIK there is nothing in here that is used in Create-UI.